ios/browser/api/history/brave_history_api_private.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_HISTORY_BRAVE_HISTORY_API_PRIVATE_H_
+#define BRAVE_IOS_BROWSER_API_HISTORY_BRAVE_HISTORY_API_PRIVATE_H_
+
+#import <Foundation/Foundation.h>
+
+#include "brave/ios/browser/api/history/brave_history_api.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+class ChromeBrowserState;
+
+OBJC_EXPORT
```

Comment:
I don't think this needs to be exported, does it? It's only used internally

Reply:
Export Removed